tests/foreman/api/test_hostgroup.py

```diff
@@ -191,6 +191,42 @@ def test_verify_bugzilla_1107708(self):
         self.assertEqual(len(host_attrs['all_puppetclasses']), 1)
         self.assertEqual(host_attrs['all_puppetclasses'][0]['name'], 'ntp')
 
+    @upgrade
+    @tier3
+    def test_rebuild_config(self):
+        """ 'Rebuild orchestration config' of an existing host group
+
+        :id: 58bf7015-18fc-4d25-9b64-7f2dd6dde425
+
+        :expectedresults: rebuild hostgroup orchestration configs successfully.
+
+        :BZ: 1686493
+
```

Comment:
As per bug, I see an issue was related to hammer CLI, where 'rebuilding hostgroup configs' was missing. Does this issue also exist in API as well?

Wanted to know are we going to add hammer CLI test for this as well?

Reply:
@omkarkhatavkar that is a RFE feature to enhance that existing feature under hammer . However this is already exists under API too ...therefore would like to cover this under API only (as so far no test exists around it)

Reply:
@vijay8451 I'm not against the adding test here or coverage (I'm really happy here). My concern was are adding right Bugzilla to test.

Reply:
@omkarkhatavkar I understand that :) ... I kept this bugzilla because I covered the same functionality  using API for hostgroup rebuild_config.
Another side I am not sure when this API was introduced  as this feature exists since long ... let me remove the `BZ` from here ..

Reply:
@omkarkhatavkar removed in latest commit .. thanks

Reply:
API was added here https://github.com/theforeman/foreman/blob/3a3eda26c981c05584a3ada48998167eb20bdd1e/app/controllers/api/v2/hostgroups_controller.rb#L105